tensorflow/python/ops/array_grad.py

```diff
@@ -448,3 +451,77 @@ def _MirrorPadGradGrad(op, grad):
 @ops.RegisterGradient("QuantizeAndDequantize")
 def _QuantizeAndDequantizeGrad(_, grad):
   return grad
+
+
+@ops.RegisterGradient("ExtractImagePatches")
+def _ExtractImagePatchesGrad(op, grad):
```

Comment:
Thanks for the contribution, this looks pretty good. Can you add a regression test to make sure the code is correct? You can use https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/conv2d_backprop_filter_grad_test.py#L25 as an example.